engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -289,7 +289,11 @@ private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: Commo
   }
 
   private def extractProperty(e: PropertyOrFieldReference, t: TypingResult): ValidatedNel[ExpressionParseError, TypingResult] = t match {
-    case Unknown => Valid(Unknown)
+    case Unknown =>
+      if(disableMethodExecutionForUnknown)
+        invalid(s"There is no property '${e.getName}' in type: ${t.display}")
```

Comment:
This message is not v. helpful as we already know that `t` is `Unknown` ;) Maybe sth like: "Property/method access on Unknown type is blocked"?

Reply:
Yeah, you are right. Already changed that, like you said.